packages/core/src/fhir-to-cda/cda-templates/commons.ts

```diff
@@ -191,21 +217,46 @@ export function formatDateToCdaTimestamp(dateString: string | undefined): string
   if (!dateString) {
     return undefined;
   }
-  const datePart = dateString.replace(/-/g, "");
-  const timePart = "000000";
-  const fractionalSeconds = "0000";
-  const cdaTimeStamp = `${datePart}${timePart}.${fractionalSeconds}`;
-  return cdaTimeStamp;
+  const formatted = dateString.replace(/-/g, "");
+  if (formatted.includes("T")) return formatted.split("T")[0];
+  return formatted;
+}
+
+export function formatDateToHumanReadableFormat(
```

Comment:
DayJs doesnt have a function for formatDateToHumanReadableFormat or formatDateToCdaTimestamp?

Reply:
Oh, good point! `formatDateToHumanReadableFormat` can be fully replaced with `dayjs(dateString).format("L LT");`